pkg/storage/export/export/export.go

```diff
@@ -766,6 +766,15 @@ func (ctrl *VMExportController) getExportPodName(vmExport *exportv1.VirtualMachi
 	return naming.GetName(exportPrefix, vmExport.Name, validation.DNS1035LabelMaxLength)
 }
 
+// getExportLabelValue will return the virtual machine's name if it is under the
+// DNS1035-specified max length, or a normalized name otherwise.
+func (ctrl *VMExportController) getExportLabelValue(vmExport *exportv1.VirtualMachineExport) string {
+	if len(vmExport.Name) <= validation.DNS1035LabelMaxLength {
```

Comment:
maybe just add a comment that this is here for backwards compatibility, otherwise looks great

Reply:
Done! I opted to put it in the logic vs. the doc comment as it _felt_ more clear to me, but I can put it anywhere if you have strong feelings otherwise.